packages/build-info/src/build-systems/nx.test.ts

```diff
@@ -30,14 +34,14 @@ test('Get the commands for a package', async (ctx) => {
   const project = new Project(ctx.fs, fixture.cwd)
   const [nx] = await project.detectBuildSystem()
 
-  expect(await nx.getCommands?.('packages/astro')).toEqual([
+  expect(await nx.getCommands?.(join('packages/astro'))).toEqual([
```

Comment:
Feels odd to be joining a single string. Is this doing path normalisation for Windows?

Reply:
jep exactly join normalizes the paths on windows and that's needed here